lib/datadog/appsec/contrib/rack/request_body_middleware.rb

```diff
@@ -24,11 +24,11 @@ def call(env)
 
             request = ::Rack::Request.new(env)
 
-            request_return, request_response = Instrumentation.gateway.push('rack.request.body', request) do
+            request_return, request_action = Instrumentation.gateway.push('rack.request.body', request) do
```

Comment:
I disagree with this change, the gateway `response` does contain an action but is a richer and different concept. In the future this return value should be an `Instrumentation::Gateway::Response` Struct (or struct-like object) instead of a basic array of `[action, event]`.

In addition this refactoring makes this file inconsistent with other places that use the instrumentation gateway.